test/_test-helpers.ts

```diff
@@ -16,8 +16,8 @@ export async function runIgnite(cmd: string, options: RunOptions = {}): Promise<
 }
 
 export async function run(cmd: string, options: RunOptions = {}): Promise<string> {
-  const pre = options.pre ? `${options.pre} ; ` : ""
-  const post = options.post ? ` ; ${options.post}` : ""
+  const pre = options.pre ? `${options.pre} && ` : ""
+  const post = options.post ? ` && ${options.post}` : ""
```

Comment:
Changed `;` output to `&&` because I was having issues on windows with the `cd` pre and post command failing due to an invalid directory, then the main run `cmd` still continuing on, which lead to indirect error messages.

This way, `run` will fail now if the pre or post command fails.